Whiskey/Tasks/PublishPowerShellScript.ps1

```diff
@@ -0,0 +1,193 @@
+
+function Publish-WhiskeyPowerShellScript
+{
+    [Whiskey.Task('PublishPowerShellScript')]
+    [Whiskey.RequiresPowerShellModule('PackageManagement', Version='1.4.7', VersionParameterName='PackageManagementVersion')]
+    [Whiskey.RequiresPowerShellModule('PowerShellGet', Version='2.2.5', VersionParameterName='PowerShellGetVersion')]
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory)]
+        [Whiskey.Context]$TaskContext,
+
+        [Parameter(Mandatory)]
+        [hashtable]$TaskParameter,
+
+        [Whiskey.Tasks.ValidatePath(Mandatory,PathType='Directory')]
+        [String] $Path,

```

Comment:
`Path` should be the path to the script to publish. Not a directory. The `ScriptManifestPath` parameter should be the `Path` parameter.